stable/cluster-autoscaler/values.yaml

```diff
@@ -39,6 +39,10 @@ extraArgs: {}
 ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
 ## affinity: {}
 
+podDisruptionBudget:
```

Comment:
If you set the default like this, you'd have to explicitly `null` out `maxUnavailable` because you can only set one of the two. I'm not even sure if that would work. Either don't set a default or do it like so:

```yaml
podDisruptionBudget: |
  maxUnavailable: 1
  # minAvailable: 2
```